napari/_qt/layer_controls/qt_layer_controls_base.py

```diff
@@ -50,7 +48,7 @@ def __init__(self, layer):
         self.setObjectName('layer')
         self.setMouseTracking(True)
 
-        self.grid_layout = LayerListGridLayout(self)
+        self.grid_layout = LayerGridLayout(self)
```

Comment:
i wonder if we can just get the layout using `self.layout()` and then drop the `grid_layout` attribute (rather than rename to `form_layout`) cc @tlambert03 (note this is purely stylistic, but might as well fix now)